app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
```

Comment:
Looks like we validate it [here](https://github.com/Katello/katello/blob/master/app/models/katello/repository.rb#L93)